### PR TITLE
Add python-gnupg recipe

### PR DESCRIPTION
**Pull request by vamega**

### CODE REVIEW

File: recipes/python-gnupg/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  run:
+    - python
+
+test:
+  imports:
+    - gnupg
+
+about:
+  home: http://pythonhosted.org/{{ name }}
```
Comment (jakirkham) [inline recipes/python-gnupg/meta.yaml]: Please just put the name here sans jinja.

**Comment by jakirkham** [comment]
Also, should we be packaging GnuPG if we are packaging a Python wrapper for it?

**Comment by vamega** [comment]
I've changed the name.
It's worth considering whether `conda-forge` should have a `gnupg` package.

However I don't think this package should depend on it.

**Comment by jakirkham** [comment]
> I've changed the name.
> It's worth considering whether `conda-forge` should have a `gnupg` package.

👍

> However I don't think this package should depend on it.

-0.5

I think this should ultimately depend on `gnupg`, but I don't feel that is enough of an issue to block this. Just that in the long run, it should make `gnupg` a dependency.

**Comment by vamega** [comment]
Let's get this merged when someone has got the time.
When we have a `gnupg` package we can add a `gnupg` dependency.

Is there a way to specify optional dependencies, since ideally that's what I think a future `gnupg` dependency should be.

**Comment by jakirkham** [comment]
SGTM. Thanks @vamega.

**Pull request merged**